apps/dialtone-documentation/docs/components/icon.md

```diff
@@ -10,6 +10,34 @@ figma_url: https://www.figma.com/file/zz40wi0uW9MvaJ5RuhcRZR/DT-Core%3A-Icons-7?
 
 Check out our complete icon collection in the [Icon Catalog](/design/icons/#icon-catalog).
 
+<code-well-header>
+    <dt-icon name="inbox" />
+</code-well-header>
+
+<code-example-tabs
+htmlCode='
+<svg
+  aria-hidden="true"
+  focusable="false"
+  data-name="[ICON_NAME]"
+  class="
+    d-icon
+    d-icon--size-[SIZE]
+    d-icon--[ICON_NAME]
+  "
+  viewBox="0 0 12 12"
+>
+  ...
+</svg>
+'
+vueCode='
+<dt-icon
+  size="500"
+  name="inbox"
+/>
+'
+showHtmlWarning />
+
```

Comment:
Think we can just remove this since there are a bunch of other vue examples already on the page.. I guess there are no html examples though hmmm.. 

@francisrupert should we leave this example code here? We've been only putting code examples on variants unless there is only a single one on the page then we put it on the main example.

Reply:
Yeah, you can remove it. The "HTML" version is technically just the raw SVG, which is only useful if you have the full source. Those can be copied off the Icon popover. 
![image](https://github.com/dialpad/dialtone/assets/1165933/11d52e13-ff46-4dea-9604-ee3f04c96f43)